signer/src/stacks/events.rs

```diff
@@ -569,6 +605,51 @@ impl RawTupleData {
             signer_bitmap: BitArray::new(bitmap.to_le_bytes()),
         }))
     }
+
+    /// This function if for transforming the print events of the
```

Comment:
nit: 
```suggestion
    /// This function is for transforming the print events of the
```
Looks like a copy pasta from my typos 😂.